qiskit/__init__.py

```diff
@@ -111,15 +115,16 @@ def __bool__(self):
     def __getattr__(self, attr):
         if not self.aer:
             try:
-                from qiskit.providers import aer
+                with warnings.catch_warnings():
+                    warnings.simplefilter("ignore")
+                    from qiskit.providers import aer
 
                 self.aer = aer.Aer
                 warnings.warn(
-                    "The qiskit.Aer entry point will be deprecated in a future release and "
-                    "subsequently removed. Instead you should use this "
-                    "directly from the root of the qiskit-aer package.",
-                    PendingDeprecationWarning,
-                    stacklevel=2,
```

Comment:
Did you drop the `stacklevel` by accident or deliberately?

Reply:
It was an accident, I"ll add it back

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/8657/commits/0a1d7a4fb81e27c47dbf713bb9ec894b94021434